manifests/w/WanCai/SmartSlides/1.0.604/WanCai.SmartSlides.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: WanCai.SmartSlides
+PackageVersion: 1.0.604
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- sppt
+ReleaseDate: 2024-04-26
+Installers:
+- Architecture: x86
+  InstallerUrl: https://files.animiz.cn/software/smartslides/smartslides_v1.0.604_x86_setup-cn.exe
+  InstallerSha256: DC640E55CAAE5D36F6F6B6BD5620AE06E1CECD4D3BEFD2BF5921578EE480877B
+  ProductCode: SmartSlides_is1
+- Architecture: x64
+  InstallerUrl: https://files.animiz.cn/software/smartslides/smartslides_v1.0.604_x64_setup-cn.exe
```

Hi @SpecterShell,

The `PackagUrl` led to a slightly different domain. Should this be used instead? 

```suggestion
  InstallerUrl: https://files.focusky.com.cn/software/smartslides/smartslides_v1.0.604_x86_setup-cn.exe
  InstallerSha256: DC640E55CAAE5D36F6F6B6BD5620AE06E1CECD4D3BEFD2BF5921578EE480877B
  ProductCode: SmartSlides_is1
- Architecture: x64
  InstallerUrl: https://files.focusky.com.cn/software/smartslides/smartslides_v1.0.604_x64_setup-cn.exe
```